extensions/trace-propagators/src/main/java/io/opentelemetry/extension/trace/propagation/TraceMultiPropagator.java

```diff
@@ -49,7 +49,7 @@
  *   .extract(context, carrier, carrierGetter);
  * }</pre>
  *
- * This class is deprecated and will be removed in 0.14.0. Users should migrate to the
+ * This class is @deprecated and will be removed in 0.14.0. Users should migrate to the
```

Comment:
I think @anuraaga was asking to add a `@deprecated` tag to the javadoc itself, not embed the `@` in the comment.